kibble/models/plans.go

```diff
@@ -48,3 +52,7 @@ func (plan *Plan) LinkPlanToPage(site *Site, PageID int) {
 		}
 	}
 }
+
+func (plan *Plan) HasExpiryDate() bool {
```

Comment:
Is this supposed to be called anywhere other than the tests?

Reply:
In the templates